Week 1 Quiz Submission from cli19-stat6250

cli19-stat6250 opened the pull request:
Updated week 1 quiz questions and answer with my own.

cli19-stat6250 committed:
Update README.md

Updated week 1 quiz questions and answer with my own.

prof-lankham reviewed (changes requested):
Existing content should *NOT* be deleted.

cli19-stat6250 committed:
Update README.md to include existing content

Existing content should NOT be deleted

cli19-stat6250 committed:
Update README.md to include existing content 2

Include existing content to md file. (I'm seeing how my GitHub timeline changes if I commit from "Review Requested").

cli19-stat6250 committed:
Merge branch 'master' into week-1

cli19-stat6250 committed:
Merge branch 'master' into week-1

cli19-stat6250 committed:
Update README.md

Re-add line that was accidentally deleted.

cli19-stat6250 committed:
Update README.md

cli19-stat6250 committed:
Update README.md

Add my questions and answers

cli19-stat6250 committed:
Update README.md
